x/farming/keeper/keeper_test.go

```diff
@@ -153,7 +157,7 @@ func (suite *KeeperTestSuite) AdvanceEpoch() {
 	suite.Require().NoError(err)
 }
 
-func (suite *KeeperTestSuite) SetFixedAmountPlan(id uint64, farmingPoolAcc sdk.AccAddress, stakingCoinWeightsMap map[string]string, epochAmountMap map[string]int64) {
+func (suite *KeeperTestSuite) CreateFixedAmountPlan(farmingPoolAcc sdk.AccAddress, stakingCoinWeightsMap map[string]string, epochAmountMap map[string]int64) {
```

Comment:
Will we have any test cases that we need to pass the `PlanType` to create a fixed amount or a ratio plan? If we do, we might as well have the function to get the plan type as an argument.

Reply:
Currently, not. But maybe we should. Let me write tests as mentioned above by @dongsam and add `PlanType` parameter if needed.

Reply:
Those are going to be added in https://github.com/tendermint/farming/issues/219

Reply:
Okay